store.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma store</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+
+    <header class="store-header">
+
+        <img class="svg-logo"
+            src="./img/karma-logo.svg" 
+            alt="Karma company logo">
+        </img>
+        <nav class="nav-bar">
+            <ul class="navigation__list">
+                <li class="active">Meet Karma</li> 
+                <li>How it Works </li>
+                <li><a href="./store.html">Store</a></li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+                
+            </div>
+        </nav>              
+    </header>
+
+  <!-- <main> -->
+
+    <main class="store-main">
+      <section class="karma-order-form">
+        <h1 class>Order your Karma wifi device today!</h1>
+        
+        <form>
+          <div class="first-last-name">
+            <fieldset>
+              <label for="fname">First name *</label><br>
+              <input type="text" id="fname" name="fname" required ><br>
+            </fieldset>
+
+            <fieldset>
+              <label for="lname">Last name *</label><br>
+              <input type="text" id="lname" name="lname" required ><br>
+            </fieldset>  
+          </div>
+
+          <div class="address">
+            <fieldset>
+              <label for="address">Address *</label><br>
+              <input type="text" id="address" name="address" required >
+            </fieldset>
+
+            <fieldset>
+              <label for="address">Address 2</label><br>
+              <input type="text" id="address" name="address">
+            </fieldset>
+          </div>
+
+
+        <div class="city-postcode"> 
+          <div class="city-dropdown">
+            <fieldset >
+              <label for="City">City*</label><br>
+              <select id="city" name="city" form="cityform" required>
+                <option value="" disabled selected hidden>Select your city...</option>
```

Comment:
great use of select and hiding the first option!

Reply:
Thanks Ellie. I googled and found it.